src/view/film-card.js

```diff
@@ -1,15 +1,25 @@
-function createFilmCardTemplate() {
+function createFilmCardTemplate(data) {
+  const {
+    name,
+    rating,
+    poster,
+    comments,
+    yearProduction,
+    duration,
+    genre,
+    description,
+  } = data;
   return `<article class="film-card">
-  <h3 class="film-card__title">The Man with the Golden Arm</h3>
-  <p class="film-card__rating">9.0</p>
+  <h3 class="film-card__title">${name}</h3>
+  <p class="film-card__rating">${rating}</p>
   <p class="film-card__info">
-    <span class="film-card__year">1955</span>
-    <span class="film-card__duration">1h 59m</span>
-    <span class="film-card__genre">Drama</span>
+    <span class="film-card__year">${yearProduction}</span>
+    <span class="film-card__duration">${duration}</span>
+    <span class="film-card__genre">${genre[0]}</span>
   </p>
-  <img src="./images/posters/the-man-with-the-golden-arm.jpg" alt="" class="film-card__poster">
-  <p class="film-card__description">Frankie Machine (Frank Sinatra) is released from the federal Narcotic Farm in Lexington, Kentucky with a set of drums and a new outlook on…</p>
-  <a class="film-card__comments">18 comments</a>
+  <img src="./images/posters/${poster}" alt="" class="film-card__poster">
+  <p class="film-card__description">${description}</p>
+  <a class="film-card__comments">${comments.length} ${comments.length >= 2 ? `comments` : `comment`}</a>
```

Comment:
обычно стараются использовать более простые сравнения, семантический смысл от этого не меняется

```suggestion
  <a class="film-card__comments">${comments.length} ${comments.length > 1 ? `comments` : `comment`}</a>
```